### PR TITLE
Setup app controllers and routes

### PR DESCRIPTION
**Pull request by kennyegun24**
## In this PR, I did the following ....
- I setup different routes to match the routes in the wireframe given
- I created my controllers manually
- I added view files for each route

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "users/:user_id/posts", to: "posts#index", as: "user_posts"
+  get "users/user_id/posts/:id", to: "posts#show", as: "user_post"
+  get "users", to: "users#index" 
+  get "users/:id", to: "users#show", as: "user"
 end
```
Comment (elisha2kyakpo1) [inline config/routes.rb:11]: ## Not a requirement
-  Job well done here, this way you will have custom URLs. Another simplest way to have your routes set up is when you use the `resources` keyword👏👏

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @kennyegun24, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description look good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly fix the broken link for the License, currently, I am getting a 404 error.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (kennyegun24) [reply config/routes.rb:11]: Hello good day, this is not a requirement, but you still invalidated my project and requested for changes?

**Review by Munsa1 — APPROVED** [review]
Hi @kennyegun24, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made requested changes :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**